router/.env.example

```diff
@@ -8,5 +8,6 @@ DEV_MODE=true
 GRAPH_API_TOKEN=<generated_with_wgc>
 CDN_URL=http://127.0.0.1:11000
 NATS_URL=nats://localhost:4222
+NATS_URL_TWO=nats://localhost:4223
```

Comment:
Please remove

Reply:
Based on our discussion, should I also remove `NATS_URL`?